.github/workflows/conda.yml

```diff
@@ -65,4 +65,4 @@ jobs:
       - uses: actions/upload-artifact@v4
         with:
           name: conda-package-${{ inputs.target }}-py${{ inputs.python-version }}
-          path: conda/package/*/scipp*.tar.bz2
+          path: conda/package/*/scipp*.conda
```

Comment:
Out of interest: Why is this different now?

Reply:
It's the new default since conda build v25.1.